.github/workflows/android-release.yml

```
@@ -27,15 +27,15 @@ jobs:
                 env:
                     RELEASE_STORE_BASE_64: ${{ secrets.RELEASE_STORE_BASE_64 }}
                 run: |
-                    echo "$RELEASE_STORE_BASE_64" | base64 --decode > release.jks
+                    printf "%s" "$RELEASE_STORE_BASE_64" | base64 --decode > release.jks
 
             -   name: Build Release AAB
```

actionlint (1.7.9)

step must run script with "run" section or run action with "uses" section

(syntax-check)